pod/perlbook.pod

```diff
@@ -54,8 +54,8 @@ programming:
 =item I<Learning Perl>  (the "Llama Book")
 
  by Randal L. Schwartz, Tom Phoenix, and brian d foy
- ISBN 978-1-4493-0358-7 [6th edition June 2011]
- ISBN 978-1-4493-0458-4 [ebook]
+ ISBN 978-1-4920-9495-1 [8th edition August 2021]
+ ISBN 978-1-4920-9492-0 [ebook]
```

Comment:
https://www.ebooks.com/en-rs/book/210327283/learning-perl/randal-l-schwartz/

Reply:
When I click on that link, I get a pop-up in my browser that says, "You're currently accessing eBooks.com Serbia but we think you're located in United States."  The corresponding U.S.-based URL is https://www.ebooks.com/en-us/book/210327283/learning-perl/randal-l-schwartz/.  How should we proceed?

Reply:
Since we don't use that URL but point to https://www.learning-perl.com/ it shouldn't matter. The ISBN for the ebook is the same - 9781492094920.